.github/workflows/comment_on_commit.yml

```
@@ -0,0 +1,16 @@
+    branches:
+      - main  # Change this to your default branch name if it's different
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment on Pull Request
+        uses: peter-evans/commit-comment@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message: '@coderabbitai review'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)